docs/pages/install.mdx

```diff
@@ -1,96 +1,48 @@
 import DownloadsComponent from "/components/DownloadsComponent";
 
-# Installing the Pelican Platform
+# Install Pelican
 
-This document explains how a user can download and install Pelican client or Pelican server components.
+This document lists the operating system requirements to install Pelican and explains how a user can download Pelican client or Pelican server components.
 
-## Installation Steps:
+> Pelican also has Docker images for users who want to serve **Pelican servers**. Refer to [Run Pelican Server using Docker image](./install/docker.mdx) for instructions.
 
+## Supported Operating Sytems
+
+Pelican supports the following operating systems:
```

Comment:
I'd be careful in making this claim. While we can technically run the pelican binary on Windows, most of the functionality is turned of or compiled out. It's not as bad with MacOS, but we're still largely untested there as well.

Reply:
Do you think is is for windows only or the claim that we _support_ the OS? If it's former, then I can add a note to Windows item suggesting that we only support client functionality. This would be the same thing for macOS where I don't think xrootd works well with mac and people should be required to build xrootd from source

Reply:
I've bugged @joereuss12 to run the `.exe` program we generated and see if that works